runtime/src/main/scala/akka/grpc/GrpcClientSettings.scala

```diff
@@ -88,9 +88,9 @@ object GrpcClientSettings {
     val port = clientConfiguration.getInt("port")
     val resolveTimeout = clientConfiguration.getDuration("service-discovery.resolve-timeout").asScala
     val sd = serviceDiscoveryMechanism match {
-      case "static" =>
+      case "static" | "grpc-dns" =>
```

Comment:
Is adding `grpc-dns` here actually needed? It looks like the `static` discovery method will already have the desired effect?

Reply:
Yes, because when enabling GRPC LB `DnsNameResolverProvider` is being used. I also wanted to reflect that in the configuration.